packages/flutter/test/material/tooltip_test.dart

```diff
@@ -3478,6 +3479,48 @@ void main() {
     );
     expect(tester.element(textAncestors.first).size, equals(tooltipConstraints.biggest));
   });
+
+  testWidgets('Tooltip does not show while transitioning from another page', (
```

Comment:
Extreme nit, for this case I think it needs a link to the issue it is checking against.
```suggestion
  // This is a regression test for https://github.com/flutter/flutter/issues/167359.
  testWidgets('Tooltip does not show while transitioning from another page', (
```

Reply:
Thanks for suggestion. Pushed the change.